packages/graphql-codegen-cli/src/utils/get-latest-version.ts

```diff
@@ -0,0 +1,12 @@
+import { fetch } from 'cross-undici-fetch';
+
+/**
+ * Fetches the version directly from the registry instead of depending on
+ * an ESM only module as latest-version does.
+ * @param packageName
+ */
+export async function getLatestVersion(packageName: string): Promise<string> {
+  return fetch(`https://unpkg.com/${packageName}/package.json`)
```

Comment:
Do we need an error handling here?
You know sometimes `fetch` gives random errors

Reply:
good idea I guess in case it errors we should log something and then skip it. We can do it in a followup PR, there wasnt error handling before. I will create an issue

Reply:
Makes sense. I didn't think it is a blocker for this PR already. It just came to my mind to make Codegen throw random errors :D